### DISCUSSION TITLE
protocol used

### DISCUSSION THREAD

**Discussion by pin32** [lavron/docker-voltronic-mqtt#3] [Q&A]
hi, I would like to know if there is the possibility of choosing the protocol used by the inverter, specifically using the "p18" and "P20".
thanks

**Comment by lavron** [comment]
Hey!
Currenly supported PI30 and PI41.
What kind of inverter do you have?

**Reply by pin32** [reply]
Voltronic vmIII end IGrid VE IV 6kW

**Comment by pin32** [comment]
the first should be fine ( I am using the container of bushrangers successfully).
the second marked EASUN uses the protocol PI18.
Could this system be made compatible with protocol 18?
thanks
